operator/src/main/resources/application.properties

```diff
@@ -9,6 +9,9 @@ kc.operator.keycloak.image=${RELATED_IMAGE_KEYCLOAK:quay.io/keycloak/keycloak:ni
 kc.operator.keycloak.image-pull-policy=Always
 kc.operator.keycloak.start-optimized=false
 kc.operator.keycloak.poll-interval-seconds=60
+# Keycloak container default requests/limits resources
+kc.operator.keycloak.resources.memory-requests=768Mi
+kc.operator.keycloak.resources.memory-limits=4Gi
```

Comment:
@vmuzikar Does this format work for you?

Reply:
@shawkins I was considering that approach, but then I chose the less verbose in terms of the Keycloak config interface (SmallRye). But I have no preference here, we can do it that way to line up with the kube approach.

Reply:
@shawkins @vmuzikar Ok, I'll amend it.